client/my-sites/scan/controller.js

```diff
@@ -97,8 +98,11 @@ export function scan( context, next ) {
 }
 
 export function scanHistory( context, next ) {
+	const state = context.store.getState();
+	const siteId = getSelectedSiteId( state );
+	const showNavigation = ! isAtomicSite( state, siteId );
```

Comment:
Just a thought that we don't want to accidentally start hiding the navigation tabs on `cloud.jetpack.com`. From my testing I couldn't make the **Scan** menu item appear for a WoA site, so maybe it's not possible. But if it ever did become possible to have a WoA site on Jetpack Cloud with the Scan menu item enabled, we wouldn't want to make unintended changes.

```suggestion
	const showNavigation = isJetpackCloud() || ! isAtomicSite( state, siteId );
```

Reply:
Yeah it make sense to safeguard. The assumption was that `isJetpackCloud()` and `isAtomic` are always mutually exclusive